database/migrations/updates/update_assets_table.php.stub

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Str;
+use Statamic\Eloquent\Assets\AssetModel;
+use Statamic\Eloquent\Database\BaseMigration as Migration;
+
+return new class extends Migration {
+    public function up()
+    {
+        Schema::table($this->prefix('assets_meta'), function (Blueprint $table) {
+            $table->string('container')->after('handle')->index();
+            $table->string('folder')->after('container')->index();
+            $table->string('basename')->after('folder')->index();
+            $table->string('filename')->after('basename')->index();
+            $table->char('extension', 10)->after('filename')->index();
+            $table->string('path')->after('extension')->index();
+            $table->jsonb('meta')->after('path')->nullable();
+
+            $table->index(['container', 'folder', 'basename']);
+        });
+
+        AssetModel::all()
+            ->each(function ($model) {
+                $model->container = Str::before($model->handle, '::');
+                $model->folder = Str::of($model->handle)->after('::')->replace('/.meta/', '')->beforeLast('/');
```

Comment:
The value saved is not correct.

Reply:
Thanks, I've updated the logic to handle root folder assets